CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Contribution guide
+
+Contributions are always welcome. The codebase is maintained using the "contributor workflow" where anyone can contribute proposals using [pull requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests).
+
+To contribution workflow is as follows:
+
+1. Fork the repository.
+2. Create a topic branch.
+3. Commit changes to the branch.
+4. Push changes to the fork.
+5. Create a pull request to merge the branch of the fork into this repository.
+6. If you had someone specifically in mind, ask them to review the pull request. 
+Otherwise, just wait for a code review: most members with merge permissions receive notifications for newly created pull requests.
+7. Address review comments, if any.
+8. Merge and submit the pull request. 
+If you don't have merge permissions, a reviewer will do it for you.
+
+> **NOTE:** Before starting a particular feature, please make sure to check if a related [issue](https://github.com/breez/breez-sdk-greenlight/issues) already exists and is not already assigned to somebody. If there is no issue created, open up a [new issue](https://github.com/breez/breez-sdk-greenlight/issues/new) if you want to first discuss and to agree on the optimal way of solving a particular problem.
+>
+> If you don't know where to start, look for issues labeled with [good first issue](https://github.com/breez/breez-sdk-greenlight/labels/good%20first%20issue).
+
+## Formatting, generating code and testing
+
+The Rust source code should be formatted according to `cargo fmt` and have no linting errors from `cargo clippy`. Any changes to public facing functions or structs should be adequately documented according to [rustdoc](https://doc.rust-lang.org/rustdoc/index.html#using-rustdoc-with-cargo). Comments on code should be written clearly and concisely, and written in English.
+
+If there are any changes to the SDK's interface, they also need to be updated in the `sdk-bindings` crate. The SDK uses [UniFFI](https://github.com/mozilla/uniffi-rs) to generate the bindings code for several different languages. Add the changes needed to the [uniffi_bindings.rs](libs/sdk-bindings/src/uniffi_binding.rs) Rust interface and also update the [UDL interface](libs/sdk-bindings/src/breez_sdk.udl) file. Whenever changes are made to the `sdk-bindings`, the [Flutter](libs/sdk-flutter) and [React Native](libs/sdk-react-native) bindings code needs to be regenerated and included in your pull request. You can regenerate these from the project root using `make codegen`. Please see the prerequisites for [Flutter](libs/sdk-flutter/README.md) and [React Native](libs/sdk-bindings/bindings-react-native/README.md) if this is your first time generating code.
```

Comment:
Although it may be a niche situation, it would be nice to also reference the possibility of having to adjust the `sdk-core/src/binding.rs` file

Reply:
Btw, a nit proposal: it may be more readable if these different files that need updates are presented in an item list instead of a large paragraph

Reply:
Look better? [db56330](https://github.com/breez/breez-sdk-greenlight/pull/1167/commits/db56330314089d3e3d48ee4ca4ed8ad147c531e0)

Reply:
Awesome :)